data/pigui/modules/settings-window.lua

```diff
@@ -340,6 +340,8 @@ local function showSoundOptions()
 	local musicLevel = Engine.GetMusicVolume()*100
 	local effectsMuted = Engine.GetEffectsMuted()
 	local effectsLevel = Engine.GetEffectsVolume()*100
+	local muteOnPause = Engine.GetMuteOnPause()
+	local muteOnFocus = Engine.GetMuteOnFocus()
```

Comment:
On a tangent: I hate boilerplate like this and would much prefer we just expose an `Engine.GetConfigOption(key, defaultValue)`, `Engine.SetConfigOption(key, value)` API with consistent (and searchable!) string keys used directly.

This is not actionable feedback within the scope of this PR, but is something you could potentially implement in a separate, later PR if you so desired.

Reply:
Not a LUA programmer so will take your lead on this. Personally I'm not so fond of string parameterisation like that since compilers/analysers cannot detect errors, which can lead to some very subtle and difficult to trace bugs.